### PR TITLE
Fix typo in thread_mattr_accessor doco [ci skip]

### PR DESCRIPTION
**Pull request by nateberkopec**
Unless he changed his name to Cavid Heinemeier Hansson

### CODE REVIEW

File: activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb [hunk 1]
```diff
@@ -111,7 +111,7 @@ def #{sym}=(obj)
   #   class Customer < Account
   #   end
   #
-  #   Customer.user = "CHH"
+  #   Customer.user = "DHH"
```
Comment (kaspth) [inline activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb]: This is meant to demonstrate that a change in Customer won't affect Account, which this change breaks. I was confused first, so we should find a better example 😁

Comment (nateberkopec) [inline activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb]: Ahhh! That makes sense. The "CH" looks a lot like "DH" in monotype fonts, I figured it had to be a typo.

Comment (rafaelfranca) [inline activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb]: lol, that is confusion. We should write something different

Comment (rafaelfranca) [inline activesupport/lib/active_support/core_ext/module/attribute_accessors_per_thread.rb]: Fixed in ff85101

**Pull request merged**